ui/src/pages/dashboard/index.module.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/scss/lib';
+
+.container {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  position: relative;
+}
+
+.sidebar {
+  width: 220px;
+  position: relative;
+  border-right: 1px solid $color-button-bg;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  .header {
+    margin: 9px 21px 0px 20px;
+    display: flex;
+    align-items: center;
+    width: 179px;
+
+    .logoImage {
+      width: 20px;
+      height: 20px;
+      border-radius: 50%;
+    }
+
+    .picContainer {
+      margin-left: auto;
+      width: 18px;
+      height: 18px;
+      display: flex;
+      flex-direction: row-reverse;
+
+      .pic {
+        border-radius: 50%;
+      }
+
+      .onlineStatus {
+        border-radius: 50%;
+        width: 8px;
+        height: 8px;
+        background: $color-online;
+        border: 1px solid $color-bg-dark;
+        position: absolute;
+        align-self: flex-end;
+        margin-top: 1px;
+        margin-left: 1px;
+      }
```

Comment:
I recommend not to unnecessarily nest classes. Until it's needed, avoid nesting styles.

Reply:
Shall I change this or its for the further application?